### PR TITLE
feat: add formulas 8.79 and 8.80 from FprEN 1992-1-1:2023 clause 8.3.2

### PR DESCRIPTION
**Pull request by egarciamendez**
Adds formulas (8.79) and (8.80) of FprEN 1992-1-1:2023 clause 8.3.2, "Internal forces due to torsion in compact or closed sections". These are the first two formulas of 8.3 on torsion and combined actions.

(8.79) gives the torsional shear stress in a wall element of a section subject to a torsional moment. (8.80) turns that stress into the shear force acting on the wall, which is what the shear rules of 8.2 need in order to design that wall.

Decisions a reviewer has to weigh:

- `T_Ed` is not listed in the "where" block of 8.3.2(2); the standard introduces it in the sentence above the formula. It is guarded with `raise_if_negative`, so the caller passes the magnitude of the torsional moment. The standard does not restrict its sign, and a negative value would return a negative stress.
- `z_i` of (8.80) is defined nowhere in text. The standard prints no "where" block under (8.80) and the symbol appears only in Figure 8.16, drawn along the centre-line of a wall. The parameter docstring says so explicitly rather than presenting the reading as if the standard stated it.
- (8.80) is printed with implicit multiplication, and (8.79) prints `2A_k`. The LaTeX uses `\cdot` in both places, matching every other formula in this repository.
- `t_eff,i` and `z_i` in (8.80) are guarded with `raise_if_negative` rather than `raise_if_less_or_equal_to_zero`: neither is a denominator, and a zero simply returns a zero force.

## Formulas as implemented

**Formula (8.79)**, `Form8Dot79TorsionalShearStressInWall`

`equation`

$$
\tau_{t,i} = \frac{T_{Ed}}{2 \cdot A_k \cdot t_{eff,i}}
$$

`complete`

$$
\tau_{t,i} = \frac{T_{Ed}}{2 \cdot A_k \cdot t_{eff,i}} = \frac{90000000.000}{2 \cdot 200000.000 \cdot 120.000} = 1.875 \ MPa
$$

`complete_with_units`

$$
\tau_{t,i} = \frac{T_{Ed}}{2 \cdot A_k \cdot t_{eff,i}} = \frac{90000000.000 \ Nmm}{2 \cdot 200000.000 \ mm^2 \cdot 120.000 \ mm} = 1.875 \ MPa
$$

**Formula (8.80)**, `Form8Dot80ShearForceInWallDueToTorsion`

`equation`

$$
V_{Ed,i} = \tau_{t,i} \cdot t_{eff,i} \cdot z_i
$$

`complete`

$$
V_{Ed,i} = \tau_{t,i} \cdot t_{eff,i} \cdot z_i = 1.875 \cdot 120.000 \cdot 400.000 = 90000.000 \ N
$$

`complete_with_units`

$$
V_{Ed,i} = \tau_{t,i} \cdot t_{eff,i} \cdot z_i = 1.875 \ MPa \cdot 120.000 \ mm \cdot 400.000 \ mm = 90000.000 \ N
$$

Contributes to #1083

## Summary by CodeRabbit

- **New Features**
  - Added calculations for torsional shear stress and torsional shear force in wall elements.
  - Added symbolic and numeric formula representations, including units and configurable precision.
  - Added validation for invalid physical input values.

- **Documentation**
  - Updated implementation status for Formulas 8.79 and 8.80.

- **Tests**
  - Added coverage for calculations, boundary cases, invalid inputs, and LaTeX output formats.

### CODE REVIEW

File: blueprints/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/formula_8_79.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+from blueprints.codes.latex_formula import LatexFormula, latex_replace_symbols
+from blueprints.type_alias import MM, MM2, MPA, NMM
+from blueprints.validations import raise_if_less_or_equal_to_zero, raise_if_negative
+
+
+class Form8Dot79TorsionalShearStressInWall(Formula):
+    r"""Class representing formula 8.79 for the calculation of the torsional shear stress in a wall element of a
+    section subject to a torsional moment.
+
+    It applies to closed thin-walled sections and solid sections, for which 8.3.2(1) allows warping torsion to be
```
Comment (rhhk-dot) [inline blueprints/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/formula_8_79.py:14]: I would formulate this differently. Clause (1) states that the warping component of torsion may normally be ignored for closed thin-walled sections and solid sections. So that may not always be the case. The general formula 8.79 applies to closed thin-walled sections and solid sections.

**Review by rhhk-dot — CHANGES_REQUESTED** [review]
Reviewed

File: tests/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/test_formula_8_80.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        [
+            (-1.875, 120.0, 400.0),  # tau_t_i is negative
+            (1.875, -120.0, 400.0),  # t_eff_i is negative
+            (1.875, 120.0, -400.0),  # z_i is negative
+        ],
+    )
+    def test_raise_error_when_invalid_values_are_given(self, tau_t_i: float, t_eff_i: float, z_i: float) -> None:
+        """Test invalid values."""
+        with pytest.raises(NegativeValueError):
+            Form8Dot80ShearForceInWallDueToTorsion(tau_t_i=tau_t_i, t_eff_i=t_eff_i, z_i=z_i)
```
[the remark this answers is not in the document]
Comment (egarciamendez) [reply tests/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/test_formula_8_80.py:40]: Added a parametrized case for t_eff_i = 0 and z_i = 0, asserting 0.0 N.

File: blueprints/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/formula_8_79.py (hunk 1 above)
Comment (egarciamendez) [reply blueprints/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/formula_8_79.py]: Reworded: 8.79 applies to closed thin-walled and solid sections. 8.3.2(1) separately says warping may normally, not always, be ignored for those.
